src/sass/time-grid.scss

```diff
@@ -135,7 +135,19 @@
   z-index: 3;
   left: 0;
   right: 0;
-  height: 1px;
+  height: 2px;
+
+  background-color: $current-time-color;
```

Comment:
these changes too? 😅

Reply:
jk the diff are just in different places

Reply:
GOD YOU SCARED ME